hivemind/compression/quantization.py

```diff
@@ -135,14 +138,15 @@ def quantize(
         except ImportError:
             raise ImportError(BNB_MISSING_MESSAGE)
 
-        quantized, (absmax, codebook) = quantize_blockwise(tensor)
+        quantized, (absmax, codebook, *extra_params) = quantize_blockwise(tensor, blocksize=4096, nested=False)
+        assert tuple(extra_params) == (4096, False, tensor.dtype, None, None)  # blocksize, nested, dtype, offset, s2
```

Comment:
Maybe we can make that tuple on the right a module-level constant? It's used twice in the code, better to make it clear we're using some predefined values

Reply:
done